src/main/resources/assets/mantle/lang/fr_fr.json

```diff
@@ -0,0 +1,26 @@
+{
+  "mantle:tooltip.cooktime": "Temps pour cuire: %ds",
+  "config.mantle.extraHeartRenderer": "Activer le rendu pour les coeurs supplémentaires",
+
+  "command.mantle.view_tags.success": "Valeurs pour l'étiquette '%s' de %s:",
+  "command.mantle.view_tags.not_found": "Valeur %s inconnue du registre '%s'",
+  "command.mantle.view_tags.no_held_block": "L'objet tenu n'est pas un bloc",
+  "command.mantle.view_tags.no_held_entity": "L'objet tenu n'est pas un oeuf d'apparition",
+  "command.mantle.view_tags.no_held_potion": "L'objet tenu n'est pas une potion",
+  "command.mantle.view_tags.no_held_fluid": "Incapable d'obtenir un fluide à partir de l'objet tenu",
+  "command.mantle.view_tags.no_held_enchantment": "L'objet tenu n'est pas enchanté",
+  "command.mantle.view_tags.no_targeted_entity": "Pas d'entité dans la zone",
+  "command.mantle.view_tags.no_targeted_tile_entity": "Pas de fluide dans la zone",
```

Comment:
Translation "dans la zone" (*in the zone*) would be more precise if switched with "à portée" (*in range*).